templates/sarif.go

```diff
@@ -0,0 +1,87 @@
+/*
+ Copyright 2024 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package templates
+
+type SarifOutput struct {
+	Version string `json:"version,omitempty"`
+	Schema  string `json:"$schema,omitempty"`
+	Runs    []Run  `json:"runs,omitempty"`
+}
+
+type Run struct {
+	Note    string   `json:"note,omitempty"`
```

Comment:
Is the output generated from this template valid when using https://sarifweb.azurewebsites.net/Validation?

I remember that there was some issue with the Note field not being in schema?

Reply:
Yeah, I have removed the note field and verified in the validator which is now succeeding.
Missed to remove this field in the template. have done in the follow up commit.